articles/per_package_documentation.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Per-Package Documentation
+permalink: articles/per_package_documentation.html
+abstract: This article describes the requirements and design of ROS 2’s per-package documentation system.
+author: Marya Belanger
+published: true
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Background
+
+ROS 2 is lacking a process for the conglomeration of package documentation.
+Discoverability for package documentation (and documentation in general) is one of the most frequent user complaints.
+
+With a standardized package documentation process that encourages package maintainers to robustly document their packages, and presents the results alongside ROS 2's general documentation, issues of discoverability and consistency will be addressed.
+
+In general, our vision for the system is:
```

Comment:
The possessive tone may not be the best way to go.  Particularly for future readers, it may not be entirely clear who it is.  I'd go more for "the vision for the system is", and so on throughout the document.